package.json

```diff
@@ -114,7 +114,7 @@
         "classnames": "^2.2.6",
         "lodash": "^4.17.15",
         "moment": "^2.24.0",
-        "office-ui-fabric-react": "6.189.4",
+        "office-ui-fabric-react": "6.199.0",
```

Comment:
I looked over the [CHANGELOG](https://github.com/OfficeDev/office-ui-fabric-react/blob/6.0/packages/office-ui-fabric-react/CHANGELOG.md) and the most notable looking changes were

* 6.198.0: Cherry pick PR #9775: Tooltip: Rendering hidden hoverable area to allow for interaction with Tooltips; and PR#9960: Tooltip: Fixing hidden hover area overlapping with Tooltip targets.
* 6.197.0: Nav: Add ability for screen reader to read selected state.
* 6.195.0: Dropdown: set aria-hidden to true on the ChevronDown icon so that it is not picked up by the screen reader
* 6.190.0: Addressing a variety of problems related to style recalculations.

Have you done any compat testing beyond the automatic E2Es?

Reply:
I've spot-checked various things such as manual/assisted assessments to see if anything looks awry - so far I haven't found any issues. It's possible I'm not aware of a certain usage - e.g. I don't think we use Navs anywhere but I could be wrong.